modules/shared_vpc/main.tf

```diff
@@ -60,6 +60,17 @@ module "project-factory" {
   skip_gcloud_download              = var.skip_gcloud_download
 }
 
+/******************************************
+  Setting API service accounts for shared VPC
+ *****************************************/
+module "shared_vpc_access" {
```

Comment:
I believe we should also modify the output for this module to
```hcl
output "project_id" {
  description = "If provided, the project uses the given project ID. Mutually exclusive with random_project_id being true."
  value       = module.shared_vpc_access.project_id
}
```
this way when this module is used, the project-id is returned only after `module.shared_vpc_access` is completed.

Reply:
+1

Reply:
done